src/services/AdvancedTradingView/adapters/baseAdapter/base.types.ts

```diff
@@ -0,0 +1,64 @@
+import { AdapterKeys, AdapterPayloadType } from '../../advancedTradingView.types'
+
+export enum LiquidityTypename {
+  burn = 'Burn',
+  mint = 'Mint',
+}
```

Comment:
I saw a variation of the enum style in this PR. All caps case is reasonable

Reply:
fixed [501266b](https://github.com/levelkdev/swapr-dapp/pull/1503/commits/501266b3e2ac4a76be7811d2fa67025ee4d1d58b)